src/backend/columnar/sql/udfs/columnar_ensure_am_depends_catalog/11.1-1.sql

```diff
@@ -0,0 +1,48 @@
+CREATE OR REPLACE FUNCTION citus_internal.columnar_ensure_am_depends_catalog()
```

Comment:
Is this the right schema?

Reply:
It's not even needed permanently; only during upgrade/downgrade I think. We can clean that up as part of a separate PR.